Section 3 - Base Features & Syntax/End/src/Person/Person.js

```diff
@@ -1,7 +1,12 @@
-import React from 'react';
+import React from "react";
 
-const person = (props) => {
-    return <p>I'm {props.name} and I am {props.age} years old!</p>;
+const person = props => {
+  return (
+    <div>
+      <p>I'm {props.name} and I am {props.age} years old!</p>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.](https://app.codacy.com/manual/saylormic/react-ultimate-guide/pullRequest?prid=4954074)